app/views/collections/_form.html.slim

```diff
@@ -8,6 +8,14 @@
       .field-element
         = f.label :description, Collection.human_attribute_name('description'), class: 'form-label'
         = f.text_area :description, class: 'form-control', rows: 10
+      .field-element
+        = f.label :visibility_level,  Collection.human_attribute_name("visibility_level"), class: 'form-label collection-edit-visibility'
```

Comment:
How about a real label here?

Reply:
You are using `human_attribute_name` here. This is a helper that makes the attribute more readable, but it's not a real label. You could also just define a new locale as we do for other attributes.

Reply:
I just didn't know. So this was a deliberate decision, fine for me.